### ISSUE TITLE
How can I know if the each row with FieldArray is dirty?

### ISSUE THREAD

**Issue by haradakunihiko** [erikras/redux-form#988]
Hello @erikras , thanks for this big update about redux@6!

I just started redux-form@6.0.0-alpha.13 and hit some difficulty with FieldArray. 
In my app, there are a lot of input data list like spread sheet. And every row has it's color with blue or white representing it's status -- whether the row is changed or not.

I guess there is no way for our 'row' to get it's data unless I use `connect` function from redux. However, I really do not want to connect all list data to the parent list component because of performance issue.

Currently, I'm doing like this with componentShouldUpdate.

```js

connect((state, ownProps) => {
  const list = state.form.myForm.list;
  return {
    rowFields = list.fields[ownProps.rowIndex],
    rowValues= list.values[ownProps.rowIndex],
    rowInitialValues = list.initial[ownProps.rowIndex]
  }
})
(Row)

```

Is there any recommendation for this situation? 

Thanks,

**Comment by haradakunihiko** [comment]
Maybe, exposing `dirty` or `value` from Field instance API would be one solution for me.

**Comment by erikras** [comment]
> Maybe, exposing dirty or value as Field instance API would be one solution for me.

That seems like a reasonable request. It's already exposing `valid` and `name`.

**Comment by erikras** [comment]
Added by #993. Will be in next release.

**Comment by haradakunihiko** [comment]
Wow, really fast! Thanks a lot!

**Comment by erikras** [comment]
Released as [`v6.0.0-alpha.14`](https://github.com/erikras/redux-form/releases/tag/v6.0.0-alpha.14).
